### DISCUSSION TITLE
Use sudo when uploading files

### DISCUSSION THREAD

**Discussion by EizEddin** [Natizyskunk/vscode-sftp#145] [Q&A] [answered]
Can we use sudo when uploading files to the remote server?
For example like when we use `--rsync-path="sudo rsync"` with rsync

**Comment by Natizyskunk** [comment, marked as answer]
@EizEddin,
Please take a look at [this section](https://github.com/Natizyskunk/vscode-sftp/blob/master/FAQ.md#how-can-i-upload-files-as-root) of the FAQ.
